component/notification/src/main/java/org/exoplatform/social/notification/Utils.java

```diff
@@ -148,15 +148,22 @@ public static void sendToStreamOwner(Set<String> receivers, String streamOwner,
    * @param activityPosterId Id of the activity poster.
    * @param posteId Id of the user who has commented.
    */
-  public static void sendToActivityPoster(Set<String> receivers, String activityPosterId, String posteId) {
+  public static void sendToActivityPoster(Set<String> receivers, String activityPosterId, String posteId , String spaceId) {
```

Comment:
Could fix th etypo of the variable posteId ?
```suggestion
  public static void sendToActivityPoster(Set<String> receivers, String activityPosterId, String posterId , String spaceId) {
```